__tests__/games.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const app = require("../games.app.js");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const db = require("../db/connection");
+
+beforeEach( () => {
+    return seed(data)
+})
+
+afterAll( () => {
+    return db.end()
+})
+
+describe ('endpoints', () =>{
+    test('paths that do not exist return 404 errors', () => {
+        return request(app)
+            .get("/api/doesnt_exist")
+            .expect(404)
+    })
+    describe('GET /api/categories', () => {
+        test ('should return an array', () => {
+            return request(app)
+            .get("/api/categories")
+            .expect(200)
+            .then((response) => {
+                expect(response.body).toEqual(
```

Comment:
toEqual would not be appropriate in this instance as the object will change in the future. Consider using matchObject or objectContaining to make this more flexible.